### DISCUSSION TITLE
How can I get the key?

### DISCUSSION THREAD

**Discussion by VladimirMonin** [logancyang/obsidian-copilot#852] [Q&A]
Hi! I really like what you're doing! I teach Python, although I consider myself a beginner. 

I really liked your plugin, and I decided to become a permanent sponsor. However, the amount is still very modest... I wanted to thank you for your contribution to the Obsidian community! And also ask how you can get the key from the Copilot plus?

**Comment by logancyang** [comment]
Please email me at logan@brevilabs.com

**Reply by VladimirMonin** [reply]
Done 🙂
